docs/user-guide/troubleshooting.rst

```diff
@@ -1,4 +1,4 @@
-***************
+**************
```

Comment:
This should have an equal number of `*` characters as letters below. I think one is missing?